build/pom.xml

```diff
@@ -0,0 +1,12 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.ibm.cohort</groupId>
+  <artifactId>build</artifactId>
+  <version>${revision}</version>
+  <name>Build Tools</name>
+  
+  <properties>
+  	<revision>0.0.1-SNAPSHOT</revision>
+  </properties>
```

Comment:
I was hoping the use of `${revision}` meant we didn't have to explicitly declare the version in this file.
I guess not having the parent pom relationship prevents us from using `${revision}` correctly.

If we're using `${revision}` as a glorified way to hardcode `0.0.1-SNAPSHOT`, then I'm fine going back to just hardcoding the value directly.

As a side note, do we care if this value isn't updated when the overall project version is updated?  I feel like it's a matter of time before someone (i.e. me) forgets to update this file.

Reply:
In fact, using `${revision}` here and in `cohort-parent/pom.xml` will mean two different things if the revision is ever changed in either file.  I guess having the build fail like that would be a nice reminder for everyone to keep this file up to date, but hardcoding it would just prevent the problem altogether.

Reply:
I think there is an advantage to the property declaration in that it can be overridden on the command line such that a -Drevision=1.2.3 will alter what artifact is built during something like a release build. This would happen for both the property in the parent pom and this pom simultaneously. 

I tried setting the parent pom and it bombed out for exactly the reason I expected. Chicken and egg. The parent wants PMD and it can't be run until the build tools are fully built and packaged.